### PR TITLE
Revert "new changes"

### PR DESCRIPTION
**Pull request by myatna-myata**
Reverts Oksana-fil/homepage#15

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
+              <div class="three wide center aligned column">
+                  <div class="ava"></div>
+              </div>
+              <div class="thirteen wide column">
+                <div class="stackable grid">
+                  <div class="sixteen wide column">
+                    <h1 class="name">Ryu Hoshi (リュウ Ryū)</h1>
                   <div class="job">Fighter into series games "Street Fighter"</div>
                   <ul class="contacts">
                     <li>
```
Comment (myatna-myata) [inline index.html:23]: !!!

File: index.html [hunk 2]
```diff
@@ -63,28 +58,25 @@ <h1 class="name">Ryu Hoshi (リュウ Ryū)</h1>
...
-          </div>
-          <div class="row">
-            <div class="ten wide column">
-              <h3>EXPERIENCE</h3>
+            <div class="row">
+              <div class="ten wide column">
+                <h3>EXPERIENCE</h3>
               <ul class="timeline">
                 <li>
                   <p><small>2018 </small></p>
```
Comment (myatna-myata) [inline index.html:82]: nice

File: index.html [hunk 3]
```diff
@@ -184,7 +176,6 @@ <h3>EXPERIENCE</h3>
                   </p>
                 </li>
               </ul>
-
               <h3>PUBLICATIONS</h3>
               <ul class="timeline">
                 <li>
```
Comment (myatna-myata) [inline index.html:181]: good

File: index.html [hunk 4]
```diff
@@ -226,9 +217,8 @@ <h3>ACHIEVEMENTS</h3>
...
                 </li>
               </ul>
-            </div>
-            <div class="six wide column">
-              <section>
+              </div>
+              <div class="six wide column">
                 <h3>Languages</h3>
                 <p>
                   <img class="mini-img" alt="us" src="https://github.githubassets.com/images/icons/emoji/unicode/1f1fa-1f1f8.png">
```
Comment (myatna-myata) [inline index.html:224]: wow

File: index.html [hunk 5]
```diff
@@ -281,13 +271,13 @@ <h3>Wants&#x1f97a;</h3>
...
-    </article>
-  </main>
-</body>
-
-</html>
\ No newline at end of file
+      </article>
+    </main>
+  </body>
+</html>
```
Comment (myatna-myata) [inline index.html]: nice

File: style/style.css [hunk 6]
```diff
@@ -1,9 +1,11 @@
 .contacts li a.twitter:hover {
     color: #1da1f2;
   }
+  
   .contacts li a.instagram:hover {
     color: #c22f97;
   }
+  
   .ava {
     content: url(../images/Ryu.png);
```
Comment (myatna-myata) [inline style/style.css:10]: good

File: style/style.css [hunk 7]
```diff
@@ -15,4 +17,4 @@
   }
   .mini-img {
     width: 0.8em;
-  }
+  }
```
Comment (myatna-myata) [inline style/style.css:20]: good

File: style/style.css (hunk 6 above)
Comment (myatna-myata) [inline style/style.css:10]: good

**Review by Oksana-fil — APPROVED** [review]

**Pull request merged**